auth/src/main/java/org/aerogear/auth/utils/TokenPersistor.java

```diff
@@ -0,0 +1,31 @@
+package org.aerogear.auth.utils;
+
+import org.aerogear.auth.credentials.TokenCredentials;
+
+/**
+ * Saves, retrieves and delete a token.
+ */
+public class TokenPersistor {
+    private TokenPersistor() {
+    }
+
+    public static TokenCredentials load() {
```

Comment:
@wei-lee what data should we pass here? Would it be a good idea to pass here the Principal?

Reply:
@ziccardi I think we don't need to pass any params here. At any time, there should be only 1 user logged into the app, the we should only keep 1 token at a time. For a new user, it should always remove the existing token and create a new one. So we just need an internal name to store the token.